subscriptions/templates/subscriptions/update.html

```diff
@@ -0,0 +1,29 @@
+{% extends "account/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block title %}{% if stripe %}{% trans "Change plan" %}{% else %}{% trans "Add plan" %}{% endif %}{% endblock %}
+
+{% block content %}
+    {% include 'account/_logged_in_sidebar.html' %}
+    <section class="account-form">
+        <h2>{% if stripe %}{% trans "Change plan" %}{% else %}{% trans "Add plan" %}{% endif %}</h2>
+
+        <form id="signup_form" method="post" action="{% url "subscription_update" %}" autocapitalize="off">
```

Comment:
What's the expected behaviour here if I've already cancelled my plan? Right now the form doesn't display any differently, but I'd expect it to maybe be disabled or at least notify me what would happen if I change plan part way through the cancellation period.